packages/@okta/vuepress-site/package.json

```diff
@@ -27,16 +27,15 @@
   },
   "dependencies": {
     "@okta/vuepress-plugin-qualtrics-feedback": "^1.0.0",
-    "bootstrap": "^4.5.3",
-    "codemirror": "^5.59.0",
-    "commander": "^2.0.0",
+    "bootstrap": "^5.1.3",
```

Comment:
looks like bootstrap v5 dropped IE11 support. Just making sure we are OK with this. Not totally sure if it would even have any effect on our site depending on what parts we are using.